src/plugins/lua/metadata_exporter.lua

```diff
@@ -356,6 +357,23 @@ local pushers = {
       timeout = rule.timeout or settings.timeout,
     }, formatted, sendmail_cb)
   end,
+  json_raw = function(task, formatted, rule)
```

Comment:
I think, it should have the name like `json_raw_tcp` to reflect what it does.

Reply:
Gack.. typo in the next commit, apologies!